client/app/stores/commentsStore.js

```diff
@@ -5,11 +5,10 @@ import reducers from '../reducers';
 import { initalStates } from '../reducers';
 
 export default props => {
-  const initialComments = props;
   const { $$commentsState } = initalStates;
   const initialState = {
     $$commentsStore: $$commentsState.merge({
-      $$comments: initialComments,
+      $$comments: props,
```

Comment:
@josiasds @justin808 Maybe you can just disable the rule for only this file or block? and not for the whole app. Don't know if the plugin will allow this?